test/integration/model/findByPks.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const chai = require('chai'),
+  Sequelize = require('../../../index'),
+  expect = chai.expect,
+  Support = require('../support'),
+  DataTypes = require('../../../lib/data-types'),
+  current = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Model'), () => {
+  beforeEach(function() {
+    this.User = this.sequelize.define('User', {
+      username: DataTypes.STRING,
+      secretValue: DataTypes.STRING,
+      data: DataTypes.STRING,
+      intVal: DataTypes.INTEGER,
+      theDate: DataTypes.DATE,
+      aBool: DataTypes.BOOLEAN,
+      binary: DataTypes.STRING(16, true)
+    });
+
+    return this.User.sync({ force: true });
+  });
+
+  describe('findByPks', () => {
+    if (current.dialect.supports.transactions) {
+      it('supports transactions', function() {
+        return Support.prepareTransactionTest(this.sequelize).then(sequelize => {
+          const User = sequelize.define('User', { 
+            id: {
+              type: Sequelize.INTEGER,
+              primaryKey: true
+            },
+            username: Sequelize.STRING 
+          });
```

Comment:
This definition of `User` is not necessary, you can just use the `this.User` you defined above